src/main.jsx

```diff
@@ -1,10 +1,10 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.jsx";
+import "./index.css";
 
-ReactDOM.createRoot(document.getElementById('root')).render(
+ReactDOM.createRoot(document.getElementById("root")).render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>,
-)
+  </React.StrictMode>
+);
```

Comment:
you have a bunch of prettier changes muddying up your git diff here. You may want to consider spending some time figuring this out so that you're not constantly getting changes. Feel free to reach out to me over Slack and perhaps I can help you with that if you need!